http-driver/pom.xml

```diff
@@ -42,5 +42,31 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+
```

Comment:
Why the use of `compile` scope for this dependencies?

Reply:
Copy&Paste error… I'll fix this.